packages/config/src/api/site_info.ts

```diff
@@ -37,30 +36,58 @@ export const getSiteInfo = async function ({
 }: GetSiteInfoOpts) {
   const { env: testEnv = false } = testOpts
 
-  if (api === undefined || mode === 'buildbot' || testEnv) {
+  if (api === undefined || testEnv || offline) {
     const siteInfo = siteId === undefined ? {} : { id: siteId }
 
-    const integrations = mode === 'buildbot' && !offline ? await getIntegrations({ siteId, testOpts, offline }) : []
-
-    return { siteInfo, accounts: [], addons: [], integrations }
+    return { siteInfo, accounts: [], addons: [], integrations: [] }
   }
 
-  const promises = [
-    getSite(api, siteId, siteFeatureFlagPrefix),
-    getAccounts(api),
-    getAddons(api, siteId),
-    getIntegrations({ siteId, testOpts, offline }),
-  ]
+  const siteInfo = await getSite(api, siteId, siteFeatureFlagPrefix)
+  const featureFlags = siteInfo.feature_flags
 
-  const [siteInfo, accounts, addons, integrations] = await Promise.all(promises)
+  const useV1Endpoint = !featureFlags?.cli_integration_installations_meta
```

Comment:
I'd be inclined to change this to `useV2Endpoint` to avoid the double boolean here

Reply:
makes sense